src/components/authorized/MyHomePage.js

```diff
@@ -6,32 +6,25 @@ import axios from 'axios';
 import Navbar from '../Navbar';
 import UserVerificationButton from '../UserVerificationButton';
 
+import CustomerCommissionList from '../shared/uicomponents/CustomerCommissionList';
+
 export default class MyHomePage extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             loading: true,
             errorMessage: '',
-            customerCommissions: [],
+            userId: '',
         };
     }
 
     componentDidMount() {
         axios.get(`${DOMAIN_URL}/api/v1/user/session`)
         .then((response) => {
-            this.setState({loading: false});
             if (response.status === 200 || response.status === 304) {
-                this.setState({ authenticated: true, user: response.data.user });
```

Comment:
what if another part of `MyHomePage` requires some other information pertaining to the user?

Reply:
true. i'll change that